evopy/evopy.py

```diff
@@ -0,0 +1,53 @@
+"""Module used for the execution of the evolutionary algorithm."""
+import numpy as np
+
+from evopy.individual import Individual
+
+
+#pylint: disable=R0913
+# we have a lot of user facing arguments, so this warning should be ignored.
+def run(fitness_function, individual_length, warm_start=None, generations=100, population_size=30,
+        num_children=1, mean=0, std=1, maximize=False):
+    """Run the evolutionary strategy algorithm.
+
+    :param fitness_function: the fitness function on which the individuals are evaluated
+    :param individual_length: the length of each individual
+    :param warm_start: the individual to start from
+    :param generations: the number of generations to execute
+    :param population_size: the population size of each generation
+    :param num_children: the number of children generated per parent individual
+    :param mean: the mean for sampling the initial population
+    :param std: the standard deviation for sampling the initial population
```

Comment:
`the mean for sampling the initial population` -> `the mean for sampling the random offsets of the initial population`

same for `std`

Reply:
Done